apps/recorder/widget.js

```diff
@@ -176,11 +176,17 @@
   };
   let getCSVHeaders = activeRecorders => ["Time"].concat(activeRecorders.map(r=>r.fields));
 
-  let writeLog = function() {
+  let writeLog = function(period) {
     entriesWritten++;
     WIDGETS["recorder"].draw();
     try {
-      var fields = [Math.round(getTime())];
+      var time = getTime();
+      if (period===1) {
+        time = Math.round(time)+"."+(Math.round(time*10))%10;
```

Comment:
@bobrippling do you think it's better to use [n.toFixed(1)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed) here like you hinted?

Reply:
Yeah I could see that being more consistent for giving a single decimal place and I believe it'll produce the same answer we're after